website/docs/e2-core/usage/deploying-e2-core.md

```diff
@@ -1,41 +1,41 @@
-# Deploying Atmo
+# Deploying E2Core
 
-Atmo is in beta, and is suitable production workloads. If you're using Atmo in production, please reach out to us via [Discord](chat.suborbital.dev) or [team@suborbital.dev](mailto:team@suborbital.dev) so we can learn how you're using it!
+E2Core is in beta, and is suitable production workloads. If you're using E2Core in production, please reach out to us via [Discord](chat.suborbital.dev) or [team@suborbital.dev](mailto:team@suborbital.dev) so we can learn how you're using it!
 
-Atmo is distributed as a Docker image: `suborbital/atmo`
+E2Core is distributed as a Docker image: `suborbital/E2Core`
 
-To run Atmo, you can mount your Runnable Bundle as a volume, build your own container image that embeds it, or set Atmo to wait for a Bundle to be uploaded.
+To run E2Core, you can mount your Runnable Bundle as a volume, build your own container image that embeds it, or set E2Core to wait for a Bundle to be uploaded.
 
 ## Volume mount
 
 To mount as a volume:
 
 ```bash
-docker run -v $(pwd):/home/atmo -e ATMO_HTTP_PORT=8080 -p 8080:8080 suborbital/atmo:latest atmo
+docker run -v $(pwd):/home/E2Core -e E2Core_HTTP_PORT=8080 -p 8080:8080 suborbital/E2Core:latest E2Core
 ```
 
-This will launch Atmo, assign it to listen on port 8080, and run in HTTP mode.
+This will launch E2Core, assign it to listen on port 8080, and run in HTTP mode.
 
 ## Embed Bundle
 
 To create your own Docker image with your Bundle embedded, you can use the Dockerfile that was created for your project by Subo:
 
 ```yaml
-FROM suborbital/atmo:latest
+FROM suborbital/E2Core:latest
 
 COPY ./runnables.wasm.zip .
 
-ENTRYPOINT atmo
+ENTRYPOINT E2Core
 ```
 
 Building this Dockerfile would result in an image that doesn't need a volume mount.
 
 ## Bundle upload
 
-To upload a Bundle after launching Atmo, use the `--wait` flag or set the
-`ATMO_WAIT=true` env var. This will cause Atmo to check the disk once per
+To upload a Bundle after launching E2Core, use the `--wait` flag or set the
+`E2Core_WAIT=true` env var. This will cause E2Core to check the disk once per
```

Comment:
Did we call this `E2Core_WAIT`, or should it be `E2CORE_WAIT`?

Reply:
There are some similar instances above (`ENTRYPOINT E2Core`, `FROM suborbital/E2Core`, `E2Core_HTTP_PORT` where I'm a little skeptical of the capitalization.

Reply:
Actually for the moment I'm trying to leave all in-code references to Atmo alone, because I don't think they've changed yet, but I must have accidentally grabbed these before I started doing each replacement manually. I'll change them back to Atmo for now.